### PR TITLE
Update to 1.6.0

### PR DESCRIPTION
**Pull request by ceball**
[EDIT: 1.6.0 now released] We have not yet released 1.6.0 as we're still testing a pre-release, but we plan to release very soon. I hope it's ok that I've started this PR now; I wanted to get feedback on a few things if possible. Please forgive me where I've failed to follow instructions - I'm new at this.

1. I've made the recipe noarch:python.

2. [EDIT: reverted, because the build failed] I've updated the build script to what's recommended at https://github.com/conda-forge/staged-recipes#7-when-or-why-do-i-need-to-use-python--m-pip-install---no-deps---ignore-installed-, instead of what's recommended at https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#script. However, I don't quite understand how that relates to the following bit from the docs: "The goal is to prevent pip from creating an egg-info directory because they do not interact well with conda". (I do understand not wanting to create an egg. I also understand that it makes sense to switch to pip. I don't know how the changed command affects the egg-info directory, though.)

3. I've added param's tests to the recipe. (In future, param's tests will likely be included in the param package, so this bit will eventually be simpler.)

4. Can I read more of the metadata from elsewhere? E.g. at least for the "about" section. The source of param's metadata is currently setup.py. The metadata in setup.py is read into param's [conda.recipe/meta.yaml](https://github.com/ioam/param/blob/master/conda.recipe/meta.yaml), which we use to generate param packages on anaconda.org's pyviz and pyviz/label/dev channels. Is it acceptable/possible to do something similar here? (In future, the source of param's metadata may change to keep up with trends in the python world, but whatever happens I'm keen for there to be just one "source of truth" for as much of the metadata as possible.)

Thanks

### CODE REVIEW

**Comment by ceball** [comment]
Re 2: I've reverted to the previous build script because all the builds failed. Next time I update the c-f param feedstock, I'll try to replicate the c-f build environment locally and test first.

**Comment by ceball** [comment]
@conda-forge-admin, please rerender.

**Comment by ceball** [comment]
1.6.0 has now been released, so I've updated to that - which means the PR is ready to review from my point of view.

**Comment by jlstevens** [comment]
Looks good to me and the tests have all passed!

**Comment by jlstevens** [comment]
@jakirkham Does this look good to you? I'm hoping to see it merged soon!

Once this version of param is released, I'll be able to follow on with the holoviews 1.10 release...

**Comment by jlstevens** [comment]
I also wonder whether this PR or the automated update PR #3 should take priority...

**Comment by jlstevens** [comment]
As I have no reason to think anything is wrong here, I'm going ahead and merging.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,18 +13,27 @@ source:
...
     - setuptools
   run:
-    - python
+    - python >=2.7
 
 test:
+  requires:
+    - nose
+  source_files:
+    - setup.cfg
```
Comment (jakirkham) [inline recipe/meta.yaml]: Hmm...interesting. Is this used to configure the tests somehow?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -13,18 +13,27 @@ source:
 
 build:
   number: 0
+  noarch: python
   script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (jakirkham) [inline recipe/meta.yaml]: So we have been moving to `pip` particularly for `noarch: python`. Not sure if there are concerns making this switch.

ref: https://conda-forge.org/docs/meta.html#use-pip

File: recipe/meta.yaml [hunk 3]
```diff
@@ -13,18 +13,27 @@ source:
...
 
 requirements:
   build:
     - python
     - setuptools
   run:
-    - python
+    - python >=2.7
```
Comment (jakirkham) [inline recipe/meta.yaml]: Probably not needed, but should be fine.

**Comment by jakirkham** [comment]
Sorry to be slow here @jlstevens. Busy week. 😩

Made a few comments above. Would suggest switching to `pip` if possible, but don't think it is urgent.

**Comment by jlstevens** [comment]
Thanks @jakirkham! All your suggested changes look good to me and we'll use them the next time we update the recipe.

**Pull request merged**